api/src/test/java/org/apache/commons/rdf/api/AbstractRDFTermFactoryTest.java

```diff
@@ -208,8 +172,8 @@ public void testCreateIRIRelative() throws Exception {
         // be possible to resolve to an absolute IRI)
         try {
             factory.createIRI("../relative");
-        } catch (UnsupportedOperationException | IllegalArgumentException ex) {
-            Assume.assumeNoException(ex);
+        } catch (IllegalArgumentException ex) {
+            Assume.assumeNoException("Relative IRIs not supported - ignore this test", ex);
```

Comment:
Agreed - perhaps a factory would support relative IRIs but would make them absolute - e.g. the factory could have its own base URI (as would be the case when parsing). Removed test.